src/main/scala/com/twitter/finagle/postgres/PostgresClient.scala

```diff
@@ -69,7 +69,7 @@ object PostgresClient {
 
   case class TypeSpecifier(receiveFunction: String, typeName: String, elemOid: Long = 0)
 
-  private[finagle] val defaultTypes = Map(
+  val defaultTypes = Map(
```

Comment:
(I say it's "fatally-bugged" because the `Future` [here](https://github.com/finagle/finagle-postgres/blob/69ab3983d6acc6aa4a8e029c96cc1cb224d6c40d/src/main/scala/com/twitter/finagle/postgres/PostgresClientImpl.scala#L82)  caches any failure to pull the typeMap at startup forever, thus borking the client)